addons/mail/models/mail_link_preview.py

```diff
@@ -33,12 +35,15 @@ def _create_from_message_and_notify(self, message):
             return self
         urls = OrderedSet(html.fromstring(message.body).xpath('//a[not(@data-oe-model)]/@href'))
         link_previews = self.env['mail.link.preview']
+        ignore_patterns = self.env["ir.config_parameter"].sudo().get_param("mail.mail_link_preview_ignore_list", "")
```

Comment:
Just added a small summary of various config parameters used in mail (in ir config parameter file), could be great to add it there. We begin to have a lot of hidden config, hence a small technical docstring helping people.